src/Angle.cpp

```diff
@@ -13,7 +13,7 @@ namespace rtt {
     }
 
     Angle::Angle(const rtt::Vector2 &vec) {
-        if (vec.length() < epsilon)
+        if (vec.length() < VECTOR_PRECISION)
```

Comment:
absolute value of length, right?

Reply:
.length() always returns a positive number :)

Reply:
my bad